examples/stats/multiple_wragged_outputs.ipynb

```diff
@@ -0,0 +1,424 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Multi-output model with wragged time measurements\n",
+    "\n",
+    "This notebook describes how to fit a model in PINTS which has multiple outputs with different measurement times for each. Additionally, we illustrate how this same framework allows different measurement models to be used for different outputs within the same forward model when doing inference.\n",
```

Comment:
This is too quick for me.
Can we change it to say "This notebook describes... for each. This situation arises when..." 1-3 lines on why this would occur, e.g. when you measure different system outputs individually.
Then leave the "additionally" for later, just add in a header and give a 2nd intro?